src/Common/src/System/Security/Cryptography/RSAOpenSsl.cs

```diff
@@ -395,6 +396,31 @@ public override void ImportParameters(RSAParameters parameters)
             ForceSetKeySize(BitsPerByte * Interop.Crypto.RsaSize(key));
         }
 
+        public override unsafe void ImportRSAPublicKey(ReadOnlySpan<byte> source, out int bytesRead)
+        {
+            fixed (byte* ptr = &MemoryMarshal.GetReference(source))
+            {
+                using (MemoryManager<byte> manager = new PointerMemoryManager<byte>(ptr, source.Length))
+                {
+                    AsnReader reader = new AsnReader(manager.Memory, AsnEncodingRules.BER);
+                    ReadOnlyMemory<byte> firstElement = reader.PeekEncodedValue();
+
+                    SafeRsaHandle key = Interop.Crypto.DecodeRsaPublicKey(firstElement.ToArray(), firstElement.Length);
```

Comment:
Similar "new overload to avoid multiple places using ToArray" comment as with the Apple version.